CHANGELOG.md

```diff
@@ -31,6 +31,8 @@
 
 ### Mimirtool
 
+* [BUGFIX] mimirtool analyze: Fix dashboard JSON unmarshalling errors by using a smaller model. #2386
```

Comment:
"using a smaller model" seems a little hard to understand from a user perspective. Maybe this could be rephrased as "using custom parsing" or something?

Reply:
```suggestion
* [BUGFIX] mimirtool analyze: Fix dashboard JSON unmarshalling errors by using custom parsing. #2386
```